sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/dll_import_export.hpp

```diff
@@ -27,14 +27,18 @@
 #if defined(_MSC_VER)
 #if defined(AZURE_SECURITY_KEYVAULT_SECRETS_BEING_BUILT)
 #define AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT __declspec(dllexport)
```

Comment:
Hmm, the `AZURE_.+DLL.*` (regexp) one should not exist. It won't work and could've never worked with DLLs, because https://github.com/Azure/azure-sdk-for-cpp/blob/fb7524874429ac89c0499716c87973c8b8ab2ca9/cmake-modules/AzureVcpkg.cmake#L115-L124

And we've just found out now.

I've found it is the case with `SECURITY_KEYVAULT_ADMINISTRATION` and `SECURITY_KEYVAULT_SECRETS`.

Please, as part of this PR, rename all these occurrences of `AZURE_.+DLL.*` into `AZ_.+DLL.*`.

Reply:
done, i renamed the ones in secrets, this PR will not rename theother ones as it will affect behavior outside of the scope of this PR